programmers/문자열-내-p와-y의-갯수.js

```diff
@@ -0,0 +1,17 @@
+function solution(s){
+    var answer = true;
+    var pcount = 0;
+    var ycount = 0;
+    for (var i in s){
```

Comment:
i라고 하기보다 element(el)같은 명확한 명칭으로 하시는게 좋습니다.
```suggestion
    for (let element in s){
```